cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -614,52 +615,39 @@ public static ResourceOperatorSupplier supplierWithMocks(boolean openShift) {
         return supplier;
     }
 
-    public static ClusterOperatorConfig dummyClusterOperatorConfig(KafkaVersion.Lookup versions, long operationTimeoutMs, String featureGates, boolean networkPolicyGeneration) {
-        return new ClusterOperatorConfig(
-                singleton("dummy"),
-                60_000,
-                operationTimeoutMs,
-                300_000,
-                false,
-                networkPolicyGeneration,
-                versions,
-                null,
-                null,
-                null,
-                null,
-                null,
-                featureGates,
-                10,
-                10_000,
-                30,
-                false,
-                1024,
-                "cluster-operator-name",
-                ClusterOperatorConfig.DEFAULT_POD_SECURITY_PROVIDER_CLASS, null);
+    public static ClusterOperatorConfig dummyClusterOperatorConfig() {
+
+        Map<String, String> envVars = new HashMap<>();
+
+        return ClusterOperatorConfig.buildFromMap(envVars, KafkaVersionTestUtils.getKafkaVersionLookup());
     }
 
     public static ClusterOperatorConfig dummyClusterOperatorConfig(KafkaVersion.Lookup versions, long operationTimeoutMs) {
-        return dummyClusterOperatorConfig(versions, operationTimeoutMs, "", true);
+        return new ClusterOperatorConfigBuilder(dummyClusterOperatorConfig(), versions)
```

Comment:
Maybe you can go through these methods and for example, remove those which are not used 10+ times or something like that? For example, this one is IMHO not used at all. Some others are used once or twice. Thanks tot he builder pattern, it should be now much easier to do this directly in the place where this is used without a need for a special method.